envoy/stats/allocator.h

```diff
@@ -84,6 +87,18 @@ class Allocator {
   virtual void forEachTextReadout(std::function<void(std::size_t)> f_size,
                                   std::function<void(Stats::TextReadout&)> f_stat) const PURE;
 
+  virtual void forEachSinkedCounter(std::function<void(std::size_t)> f_size,
```

Comment:
optional: wdyt of having `using` nicknames for all these function types? They occur a lot and are quite verbose. We may, in the future, want to change the signature and it would be slightly less annoying if we have nicknames. E.g. we might decide to pass the SharedPtr variants or we might want to have the function return a bool to indicate whether to continue. I'm not suggesting we do that now; it just would be easier to change in the future if we have the nickname.

Reply:
Done.